content/millennium/r4/diagnostic/observation.md

```diff
@@ -32,6 +32,7 @@ The following fields are returned if valued:
     * [low](https://hl7.org/fhir/R4/observation-definitions.html#Observation.referenceRange.low){:target="_blank"}
     * [high](https://hl7.org/fhir/R4/observation-definitions.html#Observation.referenceRange.high){:target="_blank"}
     * [text](https://hl7.org/fhir/R4/observation-definitions.html#Observation.referenceRange.text){:target="_blank"}
+    * [type](https://hl7.org/fhir/R4/observation-definitions.html#Observation.referenceRange.type){:target="_blank"}
```

Comment:
according to the FHIR spec: http://hl7.org/fhir/r4/observation.html
**type** should come before **text**

Reply:
https://github.com/cerner/fhir.cerner.com/pull/373/commits/102c1720e4f465083aed0e3cd865e60504cd441a